Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -97,6 +100,95 @@ def list_incidents(self, to_=None, from_=None):
                 )
             raise e
 
+    def get_host_profiles(self, params=None):
```

Comment:
please add type hints to all funcs you added, for example in this case it should be:
```python
def get_host_profiles(self, params: Optional[dict] = None) -> List[dict]:
```

Reply:
ack